### PR TITLE
Branch 21.06 databricks update [skip ci]

### PR DESCRIPTION
**Pull request by sameerz**
Update Databricks documentation for 21.06.1 release.  Remove mention of Databricks 7.3 as a supported distribution from the 21.06 release.

### CODE REVIEW

File: docs/demo/Databricks/generate-init-script-cuda11.ipynb [hunk 1]
```diff
@@ -1 +1 @@
...
....maven.org/maven2/com/nvidia/rapids-4-spark_2.12/21.06.1/rapids-4-spark_2.12-21.06.1.jar\nsudo wget -O /databricks/jars/cudf-21.06.1-cuda11.jar https://repo1.maven.org/maven2/ai/rapids/cudf/21.06.1/cudf-21.06.1-cuda11.jar\n\nsudo wget -O /etc/apt/preferences.d/cuda-repository-pin-600 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64/cuda-ubuntu1804.pin\nsudo wget -O ~/cuda-repo-ubuntu1804-11-0-local_11.0.3-450.51.06-1_amd64.deb https://developer.download.nvidia.com/compute/cuda/11.0.3/local_installers/cuda-repo-ubuntu1804-11-0-local_11.0.3-450.51.06-1_amd64.deb\nsudo dpkg -i ~/cuda-repo-ubuntu1804-11-0-local_11.0.3-450.51.06-1_amd64.deb\nsudo apt-key add /var/cuda-repo-ubuntu1804-11-0-local/7fa2af80.pub\nsudo apt-get update\nsudo apt -y install cuda-toolkit-11-0\"\"\", True)"],"metadata":{},"outputs":[],"execution_count":1},{"cell_type":"code","source":["%sh\ncd ../../dbfs/databricks/init_scripts\npwd\nls -ltr\ncat init.sh"],"metadata":{},"outputs":[],"execution_count":2},{"cell_type":"code","source":[""],"metadata":{},"outputs":[],"execution_count":3}],"metadata":{"name":"generate-init-script","notebookId":2645746662301564},"nbformat":4,"nbformat_minor":0}
```
Comment (pxLi) [inline docs/demo/Databricks/generate-init-script-cuda11.ipynb:1]: I think we do not need to release cudfjni **21.06.1**, right?

Comment (pxLi) [reply docs/demo/Databricks/generate-init-script-cuda11.ipynb:1]: my bad, we already have it. this change just make them version matched haha

**Review by viadea — APPROVED** [review]

**Review by GaryShen2008 — APPROVED** [review]

**Comment by pxLi** [comment]
build

File: docs/demo/Databricks/generate-init-script-cuda11.ipynb (hunk 1 above)
Comment (sameerz) [reply docs/demo/Databricks/generate-init-script-cuda11.ipynb:1]: We do nto need to release the cudfjni, it is already at 21.06.1 though: https://repo.maven.apache.org/maven2/ai/rapids/cudf/21.06.1/

Comment (pxLi) [reply docs/demo/Databricks/generate-init-script-cuda11.ipynb:1]: yes, I added and resolved the comment above

**Pull request merged**